app/Http/Controllers/Api/V1/LtiTool/LtiToolSettingsController.php

```diff
@@ -72,12 +73,12 @@ public function show(Organization $suborganization, $id)
     /**
      * Create Lti tool Setting
```

Comment:
Need to update docs based on https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api